bitrise.yml

```diff
@@ -3,51 +3,221 @@ default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
 
 app:
   envs:
-  - STEP_VERSION: 3.1.4
-
-  - SAMPLE_APP_URL: https://github.com/bitrise-samples/sample-apps-carthage-objc.git
+    - STEP_VERSION: 3.1.4
+    - SAMPLE_APP_URL: https://github.com/bitrise-samples/sample-apps-carthage-objc.git
+    - ORIG_BITRISE_SOURCE_DIR: $BITRISE_SOURCE_DIR
 
 workflows:
   # ----------------------------------------------------------------
   # --- workflow to Step Test
   ci:
     before_run:
-    - audit-this-step
+      - audit-this-step
+    steps:
+      - go-list:
+      - golint:
+      - errcheck:
+      - go-test:
+      - script:
+          title: Set XCODE11_12
+          inputs:
+            - content: |-
+                #!/bin/bash
+                $XCODE11_12=false
+
+                if xcodebuild -version | grep -ocE '(11|12)\..*'; then
+                  $XCODE11_12=true
+                  echo "Xcode 11 or 12 found"
+                else
+                  echo "Xcode 11 or 12 NOT found"
+                fi
+                # envman add --key XCODE11_12 --value $isXCODE11_12
+    after_run:
+      - test-without-xcconfig
+      - test-xcconfig-from-env
+      - test-xcconfig-from-input
+      - test-xcconfig-from-input-url
+      - test-cache-xcconfig-from-env
+
+  test-without-xcconfig:
+    before_run:
+      - setup_clean_repo
+    after_run:
+      - teardown
+    steps:
+      - change-workdir:
+          title: Switch working dir to test / _tmp dir
+          inputs:
+            - path: $ORIG_BITRISE_SOURCE_DIR/_tmp
+            - is_create_path: true
+      - path::./:
+          run_if: |-
```

Comment:
Should we add this to the entire workflow?

Reply:
You can only add `fun_if` to steps as far as I know.

Reply:
Right, it would be a good improvement :D